lib/article_json/export/amp/exporter.rb

```diff
@@ -16,6 +17,16 @@ def html
           end
           doc.to_html(save_with: 0)
         end
+
+        def amp_libraries
```

Comment:
can you please add some method documentation?! thanks!

Reply:
sure!